overlord/servicestate/quota_control.go

```diff
@@ -42,6 +50,54 @@ func checkSystemdVersion() {
 
 func init() {
 	checkSystemdVersion()
+	checkMemoryCGroupEnabled()
+}
+
+func checkMemoryCGroupEnabled() {
+	memoryCGroupError = memoryCGroupEnabled()
+}
+
+var CGroupsFilePath = "/proc/cgroups"
```

Comment:
this probably shouldn't be exported, it could be exported for tests via export_test.go if needed

Also, probably this should be a dirs package variable so that it lives under dirs.GlobalRootDir for easier mocking